Sofa/GL/Component/Rendering3D/src/sofa/gl/component/rendering3d/OglSceneFrame.cpp

```diff
@@ -42,6 +42,7 @@ OglSceneFrame::OglSceneFrame()
     , d_style(initData(&d_style, defaultStyle, "style", ("Style of the frame\n" + Style::dataDescription()).c_str()))
     , d_alignment(initData(&d_alignment, defaultAlignment, "alignment", ("Alignment of the frame in the view\n" + Alignment::dataDescription()).c_str()))
     , d_viewportSize(initData(&d_viewportSize, 150, "viewportSize", "Size of the viewport where the frame is rendered"))
+    , d_solidBackground(initData(&d_solidBackground, false, "solidBackground", "If true, an opaque bkacground will be rendered; otherwise the frame is rendered on top on the normal viewport."))
```

Comment:
```suggestion
    , d_solidBackground(initData(&d_solidBackground, false, "solidBackground", "If true, an opaque background will be rendered; otherwise the frame is rendered on top on the normal viewport."))
```